packages/react-loadable/index.js

```diff
@@ -0,0 +1,14 @@
+// @flow
```

Comment:
is this some leftover file? it seem that `packages/react-loadable/src/index.js` is the new entry point - and its exports are structured more nicely, reexporting things rather than assigning to the `Loadable`

also cant see `./lib/cjs` anywhere, nor any build script for it

Reply:
I'm not done